src/Symfony/Component/Mime/EmailValidatorInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Symfony\Component\Mime;
+
+interface EmailValidatorInterface
+{
+    public function isValid(string $email, mixed $emailValidation): bool;
```

Comment:
Why is this `mixed` and not `EmailValidation`, MessageIDValidation and RFCValidation are extending from? And do we really need a new Validator or would it make sense to have a way to add an own Validation?